stock_picking_report_valued/report/stock_picking_report_valued.xml

```diff
@@ -94,36 +88,18 @@
         inherit_id="stock.stock_report_delivery_has_serial_move_line"
     >
         <xpath expr="//td[@name='move_line_lot_qty_done']" position="after">
```

Comment:
suggestion: if possible, it's better to add a `name` attribute to the `th` above and find the corresponding position here in the tbody using [this trick](https://github.com/orgs/OCA/discussions/111). Otherwise, other modules that extend the same report could put some column in the middle and break the layout.

Reply:
Cool trick! I save it for the future. Thanks.
Now I fix with `name` for this case.